docs/libertyExtensions.md

```diff
@@ -41,6 +41,7 @@ The following properties are supported for server configuration.
 | jvmOptions| List | 2.0 | Inline `List` of jvm options that is written to the jvm.options file in the server directory. These properties take precedence over a specified jvm.options file.| No|
 | jvmOptionsFile| File | 2.0 | Location of the file containing JVM options to copy to the jvm.options file in the server instance.| No|
 | looseApplication | boolean | 2.0 | Indicates to install application using loose application configuration so that war or ear files do not need to be regenerated for every application update. The loose application support for ear files requires Gradle 4.0 or higher.  The default is `true`. | No |
+| mergeServerEnv | boolean | 3.0 | Merge server.env properties from a server.env specified in the `configDirectory`, server.env file specified by the `serverEnvFile` property, or from in line properties specified in the `env` attribute with the generated server.env. Conflicts are resolved with the same precedence as the replacement policy when this property is `false`. A server.env in the `configDirectory` is of lowest precedence, a `serverEnvFile` being the next highest, and `env` inline being of highest precedence.  `false`. | No |
```

Comment:
Change the `Since` version to 3.1-M1. We will change it to 3.1 once that is released.

Suggested rewording: Merge the server environment properties from all specified sources with the default generated `server.env` file in the target server. Conflicts are resolved with the same precedence as the replacement policy when this attribute is set to `false`. The properties specified in the `env` attribute are highest precedence, followed by the `serverEnvFile` attribute, then the `server.env` file located in the `configDirectory`, and finally the default generated `server.env` file in the target server. The default value is `false`.